packages/ember-toucan-core/src/components/form/controls/multiselect.ts

```diff
@@ -42,6 +42,11 @@ export interface ToucanFormMultiselectControlComponentSignature {
      */
     isReadOnly?: boolean;
 
+    /**
+     * A string to display when there are no results after filtering.
+     */
+    noResultsText: string;
```

Comment:
I'm cool with either. 

I personally favor being forward-looking with stuff like this. Because I think the chances of cleaning up no longer necessary code (for example, cleaning up the constructor assertion after most consumers have moved to Glint) are usually pretty low.